.github/workflows/flink-ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
     strategy:
       matrix:
         jvm: [8, 11]
```

Comment:
I don't know if Java 8 needs to be uninstalled.
This is because, according to the Flink 1.17 documentation, Flink 1.17 requires Java 11, not Java 8.
https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/try-flink/local_installation/

However, looking at the Dockerfile it seems Java 8 is still supported.
https://github.com/apache/flink-docker/tree/master/1.17

Reply:
Both Java 8 and 11 are supported (though 8 has been deprecated).